### PR TITLE
Add initial readme

### PR DESCRIPTION
**Pull request by arjantijms**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# Jakarta EE 10 Examples
+
+
+This repository contains code examples for Jakarta EE 10.
+
+A selection of these examples have an elaborate explanation in the Jakarta EE Tutorial.
+
+The Jakarta EE Tutorial code examples is located in a separate repository
```
Comment (kito99) [inline README.md:8]: Is this accurate? Aren't they in this repo now?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+* The test assures the response of the server/runtime is expected
+* The application is undeployed
+* The server/runtime is stopped (if needed)
+
+Focused examples use Maven for building and starting the tests and JUnit for defining and executing the tests. Arquillian is used, but only to start/stop a server, and to deploy/undeploy the test application for a given server/runtime.
+
+
+### Tutorial
+
+[Tutorial](tutorial/README.md) examples are examples that happen to be used in the current version of the tutorial. They are a mix of focused examples, and more elaborate examples. They are currently specifically a work in progress and may be moved to various other locations within this repo.
```
Comment (kito99) [inline README.md:40]: Presumably the tutorial will include some of the examples that were already in this repo or new examples (that are created in this repo), correct?

File: README.md (hunk 1 above)
Comment (arjantijms) [reply README.md:8]: It was a typo, sorry for that

File: README.md (hunk 2 above)
Comment (arjantijms) [reply README.md:40]: Yes, that is correct. The current readme is just an initial one to give people some idea of what's going on with the two folders. Basically all options are open ;)

**Pull request merged**